frontend/src/routes/AppWrapper.tsx

```diff
@@ -1,11 +1,9 @@
 import { Outlet } from 'react-router-dom';
-import NavBar from '../navigation/NavBar';
 
 // wraps all app pages
 export default function AppWrapper() {
   return (
     <>
-      <NavBar />
```

Comment:
Maybe can place the updated MainMenuBar here and delete the NavBar.tsx component since its unused. We want it here, so that users can access the Nav bar in every page they visit

Reply:
Done!